style.css

```diff
@@ -737,16 +805,548 @@ ul {
 
 #email {
   display: flex;
-  height: 48px;
-  width: 327px;
+  height: 40px;
+  width: 305px;
   padding-left: 20px;
   border-radius: 8px;
   background: #fff;
   border: 1px solid #cfd8dc;
-  font-family: "Poppins", sans-serif;
   font-size: 17px;
   font-weight: 400;
   line-height: 20px;
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+#text {
+  display: flex;
+  height: 160px;
+  width: 305px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding-left: 20px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #172d4d;
+}
+
+#contact-submit {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 48px;
+  width: 150px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  padding: 12px 16px 12px 16px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  margin-top: -20px;
+}
+
+#contact-submit:hover {
+  background-color: #6070ff;
+  font-size: 18px;
+  color: #fff;
+}
+
+#contact-submit:active {
+  background-color: #2230d2;
+  font-size: 18px;
+  color: #fff;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .ambony {
+    width: auto;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 20px;
+  }
+
+  .menu-item {
+    margin-right: 20px;
+  }
+
+  .menu-item a {
+    display: flex;
+    font-size: 1rem;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(media/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  /* work section */
+
+  .work {
+    display: grid;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 86.5%;
+    margin: 10px 0 0 28px;
+    height: auto;
+    background-color: #f4f5f7;
+  }
```

Comment:
- [ ] Good job so far implementing some of the changes. :+1: :100: , I think there still some changes you need to make here, As pointed out by the previous reviewer, there seems to be a background color in your work section which is not consistent with the Figma design, Please look into this:
- Screenshot for reference:
![Screenshot from 2023-04-08 10-56-22](https://user-images.githubusercontent.com/107632780/230715682-c11617d6-a23a-47da-a819-86419c3c94ae.png)

Reply:
Hi @V-Blaze 
I don't understand what you ask me to do. I already put: .work {background-color: #f4f5f7;}

Reply:
Hello @rbd3 , thank you for reaching out, 
- Look at the screenshot below, You can see that you have two different backgrounds (I marked one with blue, and the other with red), according to the Figma design, You are only meant to have just one background across the section:. :small_airplane: 
- SCREENSHOT:
![cr-reply](https://user-images.githubusercontent.com/107632780/230720524-ca5adff2-0f98-44ad-8dbd-fcc133d235d4.png)